app/models/reminder.rb

```diff
@@ -0,0 +1,6 @@
+class Reminder < ApplicationRecord
+  FREQUENCY_UNITS = ['day', 'week', 'month'].freeze
+
+  belongs_to :reminderable, polymorphic: true
+  belongs_to :user
```

Comment:
Also here we should add a validation on the frequency unit,  inclusion in `FREQUENCY_UNITS`